deployment/docker/Dockerfile

```diff
@@ -12,17 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG OS=linux
-ARG ARCH=amd64
 ARG GOVERSION=1.11.1
+ARG ARCH=amd64
 FROM golang:$GOVERSION-alpine as builder
+ARG OS=linux
+ARG ARCH=amd64
 WORKDIR /go/src/sigs.k8s.io/sig-storage-local-static-provisioner
 ADD . .
+RUN echo "OS: $OS ARCH: $ARCH"
```

Comment:
docker build does not show replaced variables. I add this to show current values.